tutor/commands/dev.py

```diff
@@ -41,6 +43,58 @@ def dev(context: click.Context) -> None:
     context.obj = DevContext(context.obj.root)
 
 
+@click.command(help="Configure and run Open edX from scratch, for development")
+@click.option("-I", "--non-interactive", is_flag=True, help="Run non-interactively")
+@click.option("-p", "--pullimages", is_flag=True, help="Update docker images")
+@click.pass_context
+def quickstart(context: click.Context, non_interactive: bool, pullimages: bool) -> None:
+    try:
+        utils.check_macos_docker_memory()
+    except exceptions.TutorError as e:
+        fmt.echo_alert(
+            f"""Could not verify sufficient RAM allocation in Docker:
+    {e}
+Tutor may not work if Docker is configured with < 4 GB RAM. Please follow instructions from:
+    https://docs.tutor.overhang.io/install.html"""
+        )
+
+    click.echo(fmt.title("Interactive platform configuration"))
+    config = tutor_config.load_minimal(context.obj.root)
+    if not non_interactive:
+        interactive_config.ask_questions(config, run_for_prod=False)
+    tutor_config.save_config_file(context.obj.root, config)
+    config = tutor_config.load_full(context.obj.root)
+    tutor_env.save(context.obj.root, config)
+
+    click.echo(fmt.title("Stopping any existing platform"))
+    context.invoke(compose.stop)
+
+    if pullimages:
+        click.echo(fmt.title("Docker image updates"))
+        context.invoke(compose.dc_command, command="pull")
+
+    click.echo(fmt.title("Building Docker image for LMS and CMS development"))
+    context.invoke(compose.dc_command, command="build", args=["lms"])
```

Comment:
@kdmccormick I'd like to come back to this PR. It seems to me that it's unnecessary to build the lms image in a separate command. The image should be re-built in the next start command anyway, because we launch a `docker compose up --build` command.
Am I missing something? If not, I'd like to remove this line, because it's causing unnecessary cache checks.

Reply:
@regisb When I added this, I don't think I realized that `dev start` used the compose `--build` flag. So, yes, rip it out!